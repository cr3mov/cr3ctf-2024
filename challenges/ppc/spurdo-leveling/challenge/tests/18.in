316645204183135346 945775184489601530
1 552709
1 384642
