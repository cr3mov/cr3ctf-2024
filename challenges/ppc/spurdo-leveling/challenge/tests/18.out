671177935617 671177935617 671177935617
